doc/api/python.rst

```diff
@@ -212,20 +200,18 @@ Example:
 Plotting constraints
 --------------------
 
-Contents items of type ``constraints`` are used to display one of the built-in :class:`experimental or theoretical constraints <eos.Constraint>`.
+Contents items of type ``constraints`` are used to display one of the built-in `experimental or theoretical constraints <../constraints.html>`_.
 The following keys are mandatory:
 
-``constraints`` : :class:`qualified name <eos.QualifiedName>` or list thereof
-    The name or the list of names of the constraints that will be plotted. Must identify at least one of the constraint known to EOS.
-
-``variable`` : string
-    The name of the kinematic variable to which the x axis will be mapped.
+ * ``constraints`` (:class:`QualifiedName <eos.QualifiedName>` or iterable thereof) -- The name or the list of names of the constraints
+   that will be plotted. Must identify at least one of the constraints known to EOS; see `the complete list of constraints <../constraints.html>`_.
+ * ``variable`` (*str*) -- The name of the kinematic variable to which the x axis will be mapped.
 
 When plotting multivariate constraints, the following key is also mandatory:
 
-``observable``  :class:`qualified name <eos.QualifiedName>`
-    The name of the observable whose constraints will be plotted. Must be identify one of the observables known to EOS.
-    This is only mandatory in multivariate constraints, since these can constrain more than one observable simultaneously.
+ * ``observable`` (:class:`QualifiedName <eos.QualifiedName>`) -- The name of the observable whose constraints will be plotted.
+   Must be identify one of the observables known to EOS; see `the complete list of observables <../observables.html>`_.
```

Comment:
"Must be identify" -> "Must identify" or "Must be" ?!

Reply:
Done.